### DISCUSSION TITLE
[macOS] layouts.f1mv.json and schema are saved Directly on "/Application Support" Folder instead of "/Application Support/MultiViewer for F1"

### DISCUSSION THREAD

**Discussion by tonogdlp** [f1multiviewer/issue-tracker#29] [Q&A] [answered]
Just wanted to note that the files:

   layouts.f1mv.json
   layouts.schema.json

are being saved at the 'Application Support' Folder, instead of the 'Multiviewer for F1' Folder. 
![Screen Shot 2022-06-11 at 11 19 41](https://user-images.githubusercontent.com/8254361/173200294-93245a95-28bb-4b03-90de-f96b685878b1.png)

**Comment by f1multiviewer** [comment]
Yes, this is somewhat on purpose, and helped the layouts remain when the app was renamed from F1 MultiViewer to MultiViewer for F1

**Reply by tonogdlp** [reply]
I understand, but it is a bad practice as a macOS dev.. I don't know about you, but there isn't a single file other than those 2 in my 'Application Support' folder. There are only other folders where each app puts their respective files. 

I think someone in the world can totally have an app that uses a "layouts.schema.json", and you can run into weird problems because of that. You should fix that as soon as possible, because the longer it stays there, the more difficult it will be to change it.

**Comment by f1multiviewer** [comment, marked as answer]
Alright, this will be fixed in the next release, y'all convinced me 💚
